### PR TITLE
CAT-1069 Implemented landscape mode

### PR DESCRIPTION
**Pull request by thmq**
Added WaitForTexts in NewProjectDialogTests
Removed a few solo.sleeps

### CODE REVIEW

**Comment by thmq** [comment]
ok to test

File: catroidTest/src/org/catrobat/catroid/test/content/project/ProjectManagerTest.java [hunk 1]
```diff
@@ -90,7 +90,7 @@ public void testBasicFunctions() throws NameNotFoundException, IOException {
 				getInstrumentation().getTargetContext().getPackageName(), Context.CONTEXT_IGNORE_SECURITY);
 
 		// initializeNewProject
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/test/content/project/ProjectManagerTest.java]: remove comment

File: catroidTest/src/org/catrobat/catroid/uitest/ui/activity/ProjectActivityTest.java [hunk 2]
```diff
@@ -361,6 +361,9 @@ public void testBackgroundSprite() {
 		solo.waitForText(solo.getString(R.string.new_project_dialog_title));
 
 		enterTextAndCloseDialog(sometext);
+		assertTrue("dialog not loaded in 5 seconds",
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/uitest/ui/activity/ProjectActivityTest.java]: `asserts` should be in an own section :arrow_right:  add blank line before and after `assert`

File: catroidTest/src/org/catrobat/catroid/uitest/ui/dialog/NewProjectDialogTest.java [hunk 3]
```diff
@@ -81,6 +81,9 @@ public void testNewProjectDialog() {
 		EditText newProject = (EditText) solo.getView(R.id.project_name_edittext);
 		solo.enterText(newProject, testingproject);
 		solo.clickOnButton(buttonOkText);
+		assertTrue("dialog not loaded in 5 seconds",
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/uitest/ui/dialog/NewProjectDialogTest.java]: same as above

File: catroidTest/src/org/catrobat/catroid/uitest/ui/dialog/NewProjectDialogTest.java [hunk 4]
```diff
@@ -81,6 +81,9 @@ public void testNewProjectDialog() {
...
+				solo.waitForText(solo.getString(R.string.project_orientation_title), 0, 5000));
+		solo.clickOnButton(buttonOkText);
 		solo.waitForActivity(ProjectActivity.class.getSimpleName());
 		assertEquals("New Project is not testingproject!", UiTestUtils.PROJECTNAME1, ProjectManager.getInstance().getCurrentProject().getName());
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/uitest/ui/dialog/NewProjectDialogTest.java]: assert

**Comment by avihric** [comment]
Please add some blank lines in `NewProjectDialogTest.java`, like described in the obove line comment.

**Comment by thmq** [comment]
ok to test

**Comment by thmq** [comment]
here is a test [run](https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/2295/) that worked.

**Comment by thmq** [comment]
@amix73 could you please have a look at how we have to adapt the fallbacks when in landscape mode?

**Comment by thmq** [comment]
retest this please

**Pull request merged**